eudemo/config/.gitignore

```diff
@@ -0,0 +1 @@
+*.DAT
```

Comment:
Are these coming from PROCESS? May be worth a comment just saying where we expect these to come from

Reply:
Yea coming from PROCESS. I'm a bit surprised this wasn't added before (why am I the first).